### ISSUE TITLE
Boot loop - sdmmc_common: sdmmc_init_ocr: send_op_cond (1) returned 0x107

### ISSUE THREAD

**Issue by reibuehl** [jomjol/AI-on-the-edge-device#90]
I tried to install the current firmware but the device ends up in a boot loop. 

The errors I see on the serial console are always:

```
E (4875) sdmmc_common: sdmmc_init_ocr: send_op_cond (1) returned 0x107
E (4875) connect_wlan_main: Failed to initialize the card (263). Make sure SD card lines have pull-up resistors in place.
```

I tried to re-format and re-copy the SD card contents but that did not change anything. 

I attach a full logfile:
[putty.log](https://github.com/jomjol/AI-on-the-edge-device/files/5826299/putty.log)

**Comment by reibuehl** [comment]
The SD card used was a 32GB card. Is this too big? I tried a 8GB card now and with that one it seems to work ... <confused>

**Comment by jomjol** [comment]
Yes, I made the same experience with a 64GB card. Usually 32GB should be the limit and it should work, but you need to format it in FAT32. If you use exFAT, which is needed for sure at 64GB, it will fail. I can try to update, so that exFAT is supported in one of the next releases.
I'll close this issue for the time beeing.

**Comment by reibuehl** [comment]
Could you make a note on the Installation page in the Wiki about this? That would give new users a hint what to try if the run into the same issue.

**Comment by reibuehl** [comment]
According to https://github.com/raphaelbs/esp32-cam-ai-thinker/raw/master/assets/ESP32-CAM_Product_Specification.pdf,
the official limit seems to be 4GB. 8GB and 16 GB might work too. So I don't know if exFAT support will make the situation better.

**Comment by stefan123t** [comment]
I found that the ESP32CAM modules SD Card slot was broken on my first sample.
I have a failure rate of 25% that is 1 in a lot of 4 ESP32CAMs has a broken SD Card slot.
The idea dawned when I read a post I have linked now to this upstream issue https://github.com/espressif/esp-idf/issues/965
Jason reported that he had 6 failing the SD Card slot out of his 20 modules, which is a staggering 30% failure rate.
I still have to try it with the larger capacity SD Cards, as I explicitly ordered several 8GB ones for the project now.

Thanks for your great project @jomjol everything else besides the hardware worked perfect and makes me wonder!
